assets/languages.xml

```diff
@@ -1655,16 +1655,16 @@
         <string key="gift" value="Geschenk"/>
         <string key="gift_received" value="Geschenk erhalten"/>
         <string key="confirm_sell_hexes" value="Felder verkaufen?"/>
-        <string key="confirm_skip_level" value="Skip level?"/>
+        <string key="confirm_skip_level" value="Level überspringen?"/>
         <string key="ok" value="Ok"/>
-        <string key="money" value="Money"/>
-        <string key="quantity" value="Quantity"/>
-        <string key="income" value="Income"/>
+        <string key="money" value="Geld"/>
+        <string key="quantity" value="Quantität"/>
```

Comment:
"Menge" would be better here

Reply:
I don't know how to merge my branch in [https://github.com/johanneslatzel/Antiyoy](fork) into this pull request, but I changed it there. can you help me?

Reply:
This PR is tracking `master` at johanneslatzel/Antiyoy so it's already integrated